### ISSUE TITLE
Mod Broken

### ISSUE THREAD

**Issue by Ben132231** [Vadix88/MonkeMapLoader#36]
it doesn't show any maps you can hop in too even if you have maps in the folder

Probably because the update broke it and the mod fixer doesn't have a fix for it yet

**Comment by ghost** [comment]
Yeah CI is Broken and will never be fixed Which Means MML will never work unless someone makes it Compatible with Computer++
